README.md

```diff
@@ -1,363 +1,56 @@
-
-```ruby
-index.set_settings(customRanking: ['desc(followers)'])
-```
-
-You can also configure the list of attributes you want to index by order of importance (most important first).
-
-**Note:** Algolia is designed to suggest results as you type, which means you'll generally search by prefix.
-In this case, the order of attributes is crucial to decide which hit is the best.
-
-```ruby
-index.set_settings({
-  searchableAttributes: [
-    'lastname',
-    'firstname',
-    'company',
-    'email',
-    'city',
-    'address'
-  ]}
-)
-```
-
-## Search
-
-You can now search for contacts by `firstname`, `lastname`, `company`, etc. (even with typos):
-
+Finally, you may begin searching a object using the `search` method:
 ```ruby
-# Search for a first name
-puts index.search('jimmie').to_json
-# Search for a first name with typo
-puts index.search('jimie').to_json
-# Search for a company
-puts index.search('california paint').to_json
-# Search for a first name and a company
-puts index.search('jimmie paint').to_json
-```
-
-## Search UI
-
-**Warning:** If you're building a web application, you may be interested in using one of our
-[front-end search UI libraries](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/).
-
-The following example shows how to quickly build a front-end search using
-[InstantSearch.js](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/)
-
-### index.html
-
-```html
-<!doctype html>
-<head>
-  <meta charset="UTF-8">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.1.0/themes/algolia.css" />
-</head>
-<body>
-  <header>
-    <div>
-       <input id="search-input" placeholder="Search for products">
-       <!-- We use a specific placeholder in the input to guide users in their search. -->
-    
-  </header>
-  <main>
-      
-      
-  </main>
-
-  <script type="text/html" id="hit-template">
-    
-      <p class="hit-name">
-        {}{ "attribute": "firstname" }{{/helpers.highlight}}
-        {}{ "attribute": "lastname" }{{/helpers.highlight}}
-      </p>
-    
-  </script>
-
-  <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@3.0.0"></script>
-  <script src="app.js"></script>
-</body>
-```
-
-### app.js
-
-```js
-// Replace with your own values
-var searchClient = algoliasearch(
-  'YourApplicationID',
-  'YourAPIKey' // search only API key, no ADMIN key
-);
-
-var search = instantsearch({
-  indexName: 'instant_search',
-  searchClient: searchClient,
-  routing: true,
-  searchParameters: {
-    hitsPerPage: 10
-  }
-});
-
-search.addWidget(
-  instantsearch.widgets.searchBox({
-    container: '#search-input'
-  })
-);
-
-search.addWidget(
-  instantsearch.widgets.hits({
-    container: '#hits',
-    templates: {
-      item: document.getElementById('hit-template').innerHTML,
-      empty: "We didn't find any results for the search <em>\"{{query}}\"</em>"
-    }
-  })
-);
-
-search.start();
+objects = $index.search('Fo')
```

Comment:
The `$` is php code no? Don't hesitate to run the example on the playground first.

Reply:
Yes ! Sorry, i’ve tested the wrong snippet